src/pages/Profile.jsx

```diff
@@ -6,11 +6,12 @@ import {
   Typography,
   useMediaQuery,
 } from "@mui/material";
+import { FaBlackTie, FaUserCircle } from "react-icons/fa";
 import { auth, db, storage } from "../lib/firebase";
 import { useEffect, useState } from "react";
 import { useLocation, useNavigate } from "react-router-dom";
 
-import { FaUserCircle } from "react-icons/fa";
+import { NoEncryptionTwoTone } from "@mui/icons-material";
```

Comment:
icon is not used

Reply:
I have removed 
import { NoEncryptionTwoTone } from "@mui/icons-material";